### PR TITLE
added missed step

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9786

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md [hunk 1]
```diff
@@ -200,9 +200,10 @@ Sign-in to the issuing certificate authority or management workstations with _Do
 5. On the **Subject** tab, select **Supply in the request**.
 6. On the **Cryptography** tab, validate the **Minimum key size** is **2048**.
 7. On the **Security** tab, click **Add**.
-8. Type **NDES server** in the **Enter the object names to select** text box and click **OK**.
-9. Select **NDES server** from the **Group or users names** list. In the **Permissions for** section, select the **Allow** check box for the **Enroll** permission. Clear the **Allow** check box for the **Enroll** and **Autoenroll** permissions for all other items in the **Group or users names** list if the check boxes are not already cleared. Click **OK**.
-10. Click on the **Apply** to save changes and close the console.
+8. Select **Object Types**, then, in the appeared window, choose **Computers** and click **OK**
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md:203]:
```suggestion
8. Select **Object Types**, then, in the window that appears, choose **Computers** and click **OK**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mapalko** [comment]
Checking this with some people.

**Comment by e0i** [comment]
@mapalko Please let me know when your evaluations are complete. Thanks.

**Comment by aczechowski** [comment]
I'm following up
Apologies for the delay

**Comment by vinaypamnani-msft** [comment]
I've resolved the conflict. Proposed changes look good to me.

**Pull request merged**